github/transport_test.go

```diff
@@ -45,6 +45,39 @@ func TestEtagTransport(t *testing.T) {
 	}
 }
 
+func TestUserAgentTransport(t *testing.T) {
+	ts := githubApiMock([]*mockResponse{
+		{
+			ExpectedUri: "/repos/test/blah",
+			ExpectedHeaders: map[string]string{
+				// TODO: Figure out if there is a way to make this test more dynamic
+				"User-Agent": "terraform-provider-github/4.26.1",
```

Comment:
What would be the best way for me to refer to the `packageVersion` from `transport.go` here?